dockerfiles/che-fabric8/Dockerfile

```diff
@@ -3,6 +3,12 @@
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
+ARG CHE_DASHBOARD_VERSION=next
+ARG CHE_WORKSPACE_LOADER_VERSION=next
```

Comment:
I believe we should not fall back on the next version in future, but since this approach is used in the upstream this should be ok for now